dapp-core/src/test/resources/sample_contract.groovy

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright D3 Ledger, Inc. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import io.reactivex.Observable
+import io.reactivex.disposables.Disposable
+import iroha.protocol.Commands
+import jp.co.soramitsu.dapp.AbstractDappScript
+import jp.co.soramitsu.dapp.helper.CacheManager
+import jp.co.soramitsu.iroha.java.IrohaAPI
+import jp.co.soramitsu.iroha.java.Transaction
+
+import java.security.KeyPair
+
+class TestContract extends AbstractDappScript {
+
+    private Observable<Commands.Command> observable
+
+    private Disposable disposable
```

Comment:
Why you need a class level variable? I see no accessors for it and it is not used, only set

Reply:
it is used in `close()`